Mono 2.10 build failure

`OpenRA.Mods.Common/UtilityCommands/FixClassicTilesets.cs(73,61): error CS0219: The variable 'oldLength' is assigned but its value is never used`

This breaks our automated fedora packaging, which is forced to run on a disgustingly outdated version of mono.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/10261199-mono-2-10-build-failure?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/OpenRA.Mods.Common/UtilityCommands/FixClassicTilesets.cs
+++ b/OpenRA.Mods.Common/UtilityCommands/FixClassicTilesets.cs
@@ -70,7 +70,6 @@ namespace OpenRA.Mods.Common.UtilityCommands
 						// Resize array for new entries
 						if (frames.Length > template.TilesCount)
 						{
-							var oldLength = template.TilesCount;
 							var ti = new TerrainTileInfo[frames.Length];
 							Array.Copy(tileInfo, ti, template.TilesCount);
 							tileInfoField.SetValue(template, ti);

